src/lake/Lake/CLI/Error.lean

```diff
@@ -7,6 +7,8 @@ Authors: Mac Malone
 namespace Lake
 open Lean (Name)
 
+deriving instance Repr for IO.Error
```

Comment:
I think deriving instances of built-in type classes for built-in types in `Lake` is not a good idea?

Reply:
Lake does every so often need to define additional built-in instances for built-in types, and my understanding is the preferred procedure has been to define them in Lake (and then sometimes later upstream them to core). In this case, though, this is relic of some previous code, and thus I will  simply remove it.